product-scenarios/1-manage-users-and-roles/1.1-admin-user-manage-roles/1.1.3-manage-roles-with-soap-services/pom.xml

```diff
@@ -146,10 +146,14 @@
             <artifactId>json-simple</artifactId>
             <scope>test</scope>
         </dependency>
+<!--        <dependency>-->
```

Comment:
why it is commented

Reply:
Was not needed. Will remove.

Reply:
Fixed with https://github.com/wso2/product-is/pull/14317/commits/e509054962c4a6cdd77baec91ae9c4996af02854